bindings/jupyter-modules/jupyter-ma-causal/src/components/multi-line-chart/index.js

```diff
@@ -6,84 +6,107 @@ import {schemeCategory10} from 'd3-scale-chromatic';
 import {extent as d3Extent} from 'd3-array';
 import {format as d3Format} from 'd3-format';
 
-const PADDING = {TOP: 10, BOTTOM: 20, LEFT: 20, RIGHT: 10};
-
-const getXScale = width =>
-  scaleLinear()
-    .domain([0, 1])
-    .range([PADDING.LEFT, width - PADDING.RIGHT]);
-
-const getYScale = height =>
-  scaleLinear()
-    .domain([0, 1])
-    .range([height - PADDING.BOTTOM, PADDING.TOP]);
-
 const colorScale = scaleOrdinal(schemeCategory10);
 
 export default class Chart extends Component {
   static propTypes = {
     data: PropTypes.object.isRequired,
     width: PropTypes.number.isRequired,
     height: PropTypes.number.isRequired,
+    padding: PropTypes.exact({
+      top: PropTypes.number,
+      bottom: PropTypes.number,
+      left: PropTypes.number,
+      right: PropTypes.number,
```

Comment:
nit: how about making these four required in case `scaleLiner().range()` would not handle undefined properly.

Reply:
I think these are already required, as the padding requires an exact match of the object.